### ISSUE TITLE
nuget.exe install should no-op install when using relative output directory

### ISSUE THREAD

**Issue by joelverhagen** [NuGet/Home#2511]
```
nuget.exe install Newtonsoft.Json -Version 8.0.1 -OutputDirectory myPackages -Source https://api.nuget.org/v3/index.json
nuget.exe install Newtonsoft.Json -Version 8.0.1 -OutputDirectory myPackages -Source https://api.nuget.org/v3/index.json
```

The second command should no-op since the package has already been installed. It does not. It installs again on top of the existing package.

Things work as expected when you specify no output directory or if you specify an absolute path for an output directory.

**Comment by joelverhagen** [comment]
Could be related to:
https://github.com/NuGet/Home/issues/2402

**Comment by zhili1208** [comment]
looks like this is fixed in 4.4
